### PR TITLE
Refactor Navigation Structure & Styling

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR removes the redundant wrappers from the Navigation Header component and sets the navigation to flex.

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -26,6 +26,7 @@ ul {
 }
 
 .navigation {
+  display: flex;
   margin-left: auto;
```
Comment (MagoDelBlocco) [inline src/main/webapp/style.css:30]: nit (and not the scope of this PR): use the shorthand property margin

**Review by MagoDelBlocco — APPROVED** [review]
LGTM, left a nit that is not in the scope of this PR

File: src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css:30]: I think it can be adjusted here since changes were made to the navigation's styling.

Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Replaced here; Scope of PR adjust accordingly 64a7f58

**Review by Somae — APPROVED** [review]

**Pull request merged**